src/main/java/nl/tudelft/planningstool/api/v1/CourseOccurrenceAPI.java

```diff
@@ -0,0 +1,109 @@
+package nl.tudelft.planningstool.api.v1;
+
+import nl.tudelft.planningstool.database.entities.User;
+import nl.tudelft.planningstool.database.entities.assignments.Assignment;
+import nl.tudelft.planningstool.database.entities.assignments.occurrences.CourseOccurrence;
+import nl.tudelft.planningstool.database.entities.courses.Course;
+import org.jboss.resteasy.plugins.providers.multipart.InputPart;
+import org.jboss.resteasy.plugins.providers.multipart.MultipartFormDataInput;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.List;
+import java.util.Scanner;
+import java.util.TimeZone;
+import java.util.function.Consumer;
+
+@Path("v1/courses")
+public class CourseOccurrenceAPI extends ResponseAPI {
+
+    private static final SimpleDateFormat TIME_FORMATTER = new SimpleDateFormat("yyyy-MM-dd-HH:mm");
+
+    static {
+        TIME_FORMATTER.setTimeZone(TimeZone.getTimeZone("Amsterdam"));
+    }
+
+    @PUT
```

Comment:
Because the polymer we use for uploading of files fires a put and not a post. We didnt have to write the logic ourselves, but have to deal with the put :p

I will be contacting the owner anyways for other improvements and this is one of the topics :)